docs/src/app/components/pages/components/tabs.jsx

```diff
@@ -6,10 +6,14 @@ let { Colors, Typography } = Styles;
 let Code = require('tabs-code');
 
 
-class TabsPage extends React.Component {
+export default class TabsPage extends React.Component {
 
-  constructor() {
-    super();
+  static contextTypes = {
+    router: React.PropTypes.func,
+  };
```

Comment:
This needed anymore? Haven't been keeping up with changes, but I saw some router changes and other files changing contextTypes